static/aie.config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What does this file do? Is it still for testing?

Reply:
removed this file in newest push - not sure at what point it got committed it isn't a file I've ever messed with

Reply:
I don't think this file has been removed